install-and-launch.sh

```diff
@@ -429,6 +431,37 @@ launch_sublime() {
 }
 
 install_sublime() {
+    if [ -z "$clone_platform" ]; then
```

Comment:
I wanted to move this up to the top so that all users will have a `sublime-platform` directory from which they can edit `sublime.env` and `certbot.env`, even if they exit early to perform SSL manual steps.